README.md

```diff
@@ -1,43 +1,58 @@
 # LLMOps Pipeline
 
-This project showcase LLMOps pipeline that fine-tune a small sized LLM model to prepare the outage of the service type LLM. For this project, we choose [Gemini 1.0 Pro](https://deepmind.google/technologies/gemini/) for service type LLM and [Gemma](https://blog.google/technology/developers/gemma-open-models/) 2B/7B for small sized LLM model.
+This project showcase LLMOps pipeline that fine-tune a small size LLM model to prepare the outage of the service LLM. For this project, we choose [Gemini 1.0 Pro](https://deepmind.google/technologies/gemini/) for service type LLM and [Gemma](https://blog.google/technology/developers/gemma-open-models/) 2B/7B for small sized LLM model.
```

Comment:
```suggestion
This project showcases LLMOps pipeline that fine-tunes a small sized LLM model to prepare the outage of the service LLM. For this project, we choose [Gemini 1.0 Pro](https://deepmind.google/technologies/gemini/) for service type LLM and [Gemma](https://blog.google/technology/developers/gemma-open-models/) 2B/7B for small-sized LLM.
```

What's an outage? Also, we should make it clear how the small-sized LLM is leveraged and how the service LLM is leveraged.